front-end/src/App.js

```diff
@@ -35,22 +35,6 @@ class App extends Component {
     );
   }
 
-  componentWillMount () {
```

Comment:
Could you explain briefly why you removed this code? Is it no longer needed?

Reply:
This is dublicated code from 'withAuth' function. I didn't noticed it after first attempts, without authorization wrapper component. It's not needed